app/views/govuk_publishing_components/components/_radio.html.erb

```diff
@@ -3,91 +3,109 @@
   items ||= []
 
   label ||= nil
+  heading ||= nil
+  is_page_heading ||= false
   hint ||= nil
   error_message ||= nil
   error_items ||= nil
+
   has_error = error_message || error_items&.any?
-  hint_id = "hint-#{SecureRandom.hex(4)}"
+  hint_id = "hint-#{SecureRandom.hex(4)}" if hint
   error_id = "error-#{SecureRandom.hex(4)}"
 
   form_group_css_classes = %w(govuk-form-group)
   form_group_css_classes << "govuk-form-group--error" if has_error
 
+  aria = "#{hint_id} #{"#{error_id}" if has_error}".strip if hint or has_error
+
   # check if any item is set as being conditional
   has_conditional = items.any? { |item| item.is_a?(Hash) && item[:conditional] }
 %>
 <%= content_tag :div, class: form_group_css_classes do %>
+  <%= tag.fieldset class: "govuk-fieldset", "aria-describedby": aria do %>
 
-  <% if hint %>
-    <%= render "govuk_publishing_components/components/hint", {
-      id: hint_id,
-      text: hint
-    } %>
-  <% end %>
+    <% if heading.present? %>
```

Comment:
Bit pedantic but can the indenting match what it was? It seems to have gained two spaces which is making this diff confusing because blocks without changes are showing up as different.

Reply:
Yeah, appreciate that it's confusing. I had to add a fieldset element to wrap the radios so I think the indenting is correct, unless I've missed something? I think you can set the PR to ignore whitespace changes when reviewing changes?

Reply:
Ah, yep, it's the fieldset that's doing it. Sorry, missed that. Ignore this comment.